sdk/containerregistry/azure-containerregistry/azure/containerregistry/_generated/operations/_authentication_operations.py

```diff
@@ -184,6 +242,8 @@ def exchange_acr_refresh_token_for_acr_access_token(
         :type scope: str
         :param refresh_token: Must be a valid ACR refresh token.
         :type refresh_token: str
+        :param api_version: Api Version. The default value is "2021-07-01".
```

Comment:
are there multiple api versions or only one right now

Reply:
We'll support passing service version. Related issue: https://github.com/Azure/azure-sdk-for-python/issues/18840

Reply:
ok sweet so it's not being addressed in this pr right?

Reply:
Correct, it will be ready for Jan. release.

Reply:
I think we want to have the feature implemented in RC too so we we GA there will be no changes other than small changes like version update.  Is passing service version not planned for Python?

The service supports two versions now, one ealier preview version, and 2021-07-01 (current latest)

Reply:
What I am concerned is about the time. Python has passed the original releasing date, so I prefer to have a release with API updates first.